apps/formbricks-com/components/shared/UseCaseCTA.tsx

```diff
@@ -13,9 +13,9 @@ export default function UseCaseHeader({ href }: UseCaseCTAProps) {
       <Button variant="secondary" href={href}>
         Step-by-step manual
       </Button>
-      <div className="space-y-1 text-center">
+      <div className="space-y-1 text-center ">
         <Button
-          variant="darkCTA"
+          className="bg-gray-800 text-gray-500 hover:text-white"
```

Comment:
Can you make the text more lighter for the dark mode? It looks really off currently. And can you also handle the text coloring being dynamic as per the light/dark mode as well? The text isn't really visible in the light mode:
![image](https://github.com/formbricks/formbricks/assets/55556994/07b934d9-7077-4ad7-ab07-88593a53c203)